drivers/staging/renesas/rcar/ddr/ddr_a/ddr_init_e3.c

```diff
@@ -1046,7 +1046,7 @@ uint32_t recovery_from_backup_mode(void)
    } else {
       NOTICE("[COLD_BOOT]");
    } /*  ddrBackup */
-   err=dram_update_boot_status(ddrBackup);
+   err=rcar_dram_update_boot_status(ddrBackup);
```

Comment:
nit: space needed around `=`.

Reply:
This file doesn't follow at all the kernel coding style, there is no point in fixing specific problems here. It doesn't even use tabs for indentation.